xc7/yosys/synth.tcl

```diff
@@ -1,14 +1,22 @@
 yosys -import
+echo on
```

Comment:
I assume this was for debugging, remove?

Reply:
Same as above, corrected.